Project/src/server/Room.java

```diff
@@ -32,7 +32,12 @@
 	private final static String SAVEMUTE = "savemute";
 	private final static String LOADMUTE = "loadmute";
 
```

Comment:
I just realized I forgot to add how saveMute() and loadMute() are called. They are called inside the switch case where every command is processed, taking in just the ServerThread client as the parameter.